micronaut-camunda-bpm-feature/src/main/java/info/novatec/micronaut/camunda/bpm/feature/AdminUserCreator.java

```diff
@@ -80,9 +81,16 @@ protected boolean adminGroupAlreadyExists() {
     protected void createUser() {
         User newUser = identityService.newUser(adminUser.getId());
         newUser.setPassword(adminUser.getPassword());
-        newUser.setFirstName(adminUser.getFirstname());
-        newUser.setLastName(adminUser.getLastname());
-        adminUser.getEmail().ifPresent(newUser::setEmail);
+
+        newUser.setFirstName(adminUser.getFirstname()
```

Comment:
sorry, I think it is more readable if we write all statements in a single line and also remove the empty lines